etc/configure.py

```diff
@@ -12,9 +12,9 @@
 To use, create a configuration directory tree that contains any of these:
 
 * Requirements files named with this convention:
- - requirements_base.txt contains common requirements installed on all platforms.
- - requirements_win.txt, requirements_linux.txt, requirements_mac.txt and
-   requirements_posix.txt are os-specific requirements.
+ - requirements-py36_base.txt contains common requirements installed on all platforms.
```

Comment:
What's the difference between a requirements files for 3.6, 3.7 and 3.8?

Reply:
Requirements files difer in wheels that have native code  like intbitset is diff for py36,py37,py38 and many such more wheels like extractcode-7z